client/src/app/site/pages/meetings/pages/motions/pages/motion-detail/components/motion-meta-data/motion-meta-data.component.html

```diff
@@ -293,7 +293,8 @@ <h4>{{ (motion.hasLeadMotion ? 'Identical amendments' : 'Identical motions') | t
             [routerLink]="['/', activeMeetingId, 'motions', identicalMotion.sequential_number]"
             [state]="{ back: 'true' }"
         >
-            {{ identicalMotion.numberOrTitle }}
+            {{ identicalMotion.numberOrTitle
+            }}{{ showIdenticalMotionComma(identicalMotion, motion?.identical_motions) ? ',' : '' }}
```

Comment:
Please take a look at the `CommaSeparatedListingComponent` and exactly copy the behaviour from there:

https://github.com/OpenSlides/openslides-client/blob/1d78752728118b54fce22a62fea235955bd8139e/client/src/app/ui/modules/comma-separated-listing/components/comma-separated-listing/comma-separated-listing.component.html#L1-L8

You can use the `let isLast = last` syntax in combination with `ngIf` to determine whether or not to place a comma. Also, use the same special space ( `&ensp;`) to add spacing between the items.

Reply:
Nevermind, you can simply use the component directly in combination with an `ng-template` - I overlooked this earlier, this is way better than copying the behaviour

Reply:
See my second comment

Reply:
Yes, using the component is a better idea. Add a commit with the `comma-seperated-listing-component`.